share/nitweb/stylesheets/nitweb.css

```diff
@@ -90,6 +122,89 @@ a {
     background-color: #ff9c0f;
 }
 
+/* forms */
+
+.has-icon {
+    position: relative;
+}
+
+.has-icon .form-control {
+	padding-left: 35px;
+}
+
+.form-control-icon {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 2;
+    display: block;
+    width: 34px;
+    height: 34px;
+    line-height: 34px;
+    text-align: center;
+    pointer-events: none;
+}
+
+/* search */
+
+.search-input {
+	width: 100%;
+}
+
+.search-results {
+	position: absolute;
+	right: 0;
+}
+
+.search-results .card.active {
+	background: #eee;
+	border-color: #eee;
+}
+
+/* navs */
+
+.nav-tabs li { cursor: pointer; }
+
+.navbar-fixed-top {
+	background-color: #1E9431;
+	box-shadow: 0 0 4px rgba(0,0,0,.14),0 4px 8px rgba(0,0,0,.28);
+}
+
+.navbar-fixed-top .form-control:hover, .navbar-fixed-top .form-control:focus {
+	background: rgba(255, 255, 255, 0.2);
+}
+
+.navbar-fixed-top .form-control {
+	background: rgba(255, 255, 255, 0.1);
+    border: none;
+    color: #fff;
+    box-shadow: none;
+}
+
+.navbar-fixed-top .form-control-icon {
+	color: #fff;
+}
+
+.navbar-fixed-top *::-webkit-input-placeholder {
+    color: #fff;
+}
+.navbar-fixed-top *:-moz-placeholder {
+    /* FF 4-18 */
+    color: #fff;
+}
+.navbar-fixed-top *::-moz-placeholder {
+    /* FF 19+ */
+    color: #fff;
+}
+.navbar-fixed-top *:-ms-input-placeholder {
+    /* IE 10+ */
+    color: #fff;
+}
```

Comment:
Yeah but it's css, I don't care.
On Jun 3, 2016 09:17, "jcbrinfo" <notifications@github.com> wrote:

> In share/nitweb/stylesheets/nitweb.css
> <https://github.com/nitlang/nit/pull/2157#discussion_r65704976>:
>
> > +
> > +.navbar-fixed-top *::-webkit-input-placeholder {
> > +    color: #fff;
> > +}
> > +.navbar-fixed-top *:-moz-placeholder {
> > +    /* FF 4-18 */
> > +    color: #fff;
> > +}
> > +.navbar-fixed-top *::-moz-placeholder {
> > +    /* FF 19+ */
> > +    color: #fff;
> > +}
> > +.navbar-fixed-top *:-ms-input-placeholder {
> > +    /* IE 10+ */
> > +    color: #fff;
> > +}
>
> Why don’t you group all these rules with a comma? DRY
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nitlang/nit/pull/2157/files/a9a7d417eb81514de348d4c23cd0ce67dab0e886#r65704976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAjl6O2hA53A5KDSDOlWoEUgKozeGODiks5qICmHgaJpZM4ItE5_>
> .
>